lib/Mail.php

```diff
@@ -553,6 +553,8 @@ public function sendWithoutRendering(MailerInterface $mailer = null): static
                 $recipients = $this->getDebugMailRecipients($recipients);
             }
 
+            \Pimcore::getEventDispatcher()->dispatch($event, MailEvents::PRE_LOG);
```

Comment:
I'm not sure if we really should re-use the same object here :thinking:

Reply:
It's that same pattern as always within event dispatcher. Why would you do it different in this case?

Reply:
@jdreesen I agree with @MarkTro